service.go

```diff
@@ -299,7 +299,6 @@ func NewService(serviceName string, serviceVersion string, confProfile string, c
 		common.Discovery = casted
 	} else {
 		common.Discovery = nil
-		common.LoggingClient.Info("ProtocolDiscovery is not implemented")
```

Comment:
remove because `LoggingClient` is not yet initialized here which will cause panic